lib/tasks/review_profile.rake

```diff
@@ -41,17 +45,36 @@ namespace :users do
             sp_name: nil,
           )
 
+          success = true
           STDOUT.puts "User's profile has been activated and the user has been emailed."
         else
-          STDOUT.puts "There was an error activating the user's profile. Please try again"
+          error = "There was an error activating the user's profile. Please try again"
         end
       else
-        STDOUT.puts 'User is past the 30 day review eligibility'
+        error = 'User is past the 30 day review eligibility'
+      end
+    rescue StandardError => e
+      success = false
+      exception = e
+      raise e
+    ensure
+      analytics_error_hash = nil
+      if error.present?
+        STDOUT.puts error
+        analytics_error_hash = { message: error }
       end
+
+      Analytics.new(
+        user: AnonymousUser.new, request: nil, session: {}, sp: nil,
```

Comment:
Does the user uuid get logged if we use `AnonymousUser`?

Reply:
no, I'd try to pull the user by UUID if possible for these

Reply:
Whoops, yeah that is supposed to be `user`. Willfix